### PR TITLE
Update Changelog for 2.13.2

### PR DESCRIPTION
**Pull request by lawmicha**
Moved the AWSMobileClient and Pinpoint changes under 2.13.2

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,10 @@
 # AWS Mobile SDK for iOS CHANGELOG
-## Unreleased Changes
-- Added support for adding custom demographic to pinpoint [PR: #2410](https://github.com/aws-amplify/aws-sdk-ios/pull/2410)
 
-## Unreleased
+## 2.13.2
 
 ### New features
 - **AWSMobileClient** Added IdentityProvider strings for Sign in with Apple: `IdentityProvider.apple`. See [Issue #1809](https://github.com/aws-amplify/aws-sdk-ios/issues/1809)) and [PR #2425](https://github.com/aws-amplify/aws-sdk-ios/pull/2425).
```
Comment (royjit) [inline CHANGELOG.md:6]: Remove extra `)`

Comment (royjit) [inline CHANGELOG.md:6]: Looks like our previous pr has subsection for each service/type. So maybe change to this:

```
### New features
- **AWSMobileClient**
  - Added IdentityProvider strings for Sign in with Apple: `IdentityProvider.apple`. See [Issue #1809](https://github.com/aws-amplify/aws-sdk-ios/issues/1809)) and [PR #2425](https://github.com/aws-amplify/aws-sdk-ios/pull/2425).

- AWS Pinpoint
 - Added support for adding custom demographic to pinpoint [PR: #2410](https://github.com/aws-amplify/aws-sdk-ios/pull/2410)
```

Comment (royjit) [reply CHANGELOG.md:6]: Should we move the line `Added support for Pinpoint Journey Push Notifications` to New Features?

Comment (lawmicha) [reply CHANGELOG.md]: > Should we move the line `Added support for Pinpoint Journey Push Notifications` to New Features?

good idea, let me find the PR as well

**Pull request merged**